docs/UserGuide.md

```diff
@@ -461,11 +491,11 @@ Notify the user if a certain stock is below threshold
 
 | Action    | Format, Examples                                                                                    |
 |-----------|-----------------------------------------------------------------------------------------------------|
-|**Add to Inventory**    | `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]...​` <br> e.g., `add n/Chicken q/3 s/ShengSiong t/Poultry` |
+|**Add to Inventory**    | `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [metric/METRIC] [t/TAG]...​` <br> e.g., `add n/Chicken q/3 s/ShengSiong t/Poultry` |
 |**Clear from Inventory**  | `clear-i`            |
 |**Delete from Inventory** | `delete-i INDEX`<br> e.g., `delete 3`         |
 |**Edit Inventory**   | `edit-i INDEX [n/NAME] [q/QUANTITY] [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]…​`<br> e.g.,`edit 1 n/Chicken q/50`    |
-|**Find in Inventory**   | `find-i PREFIX KEYWORD [MORE_KEYWORDS]`<br> e.g., `find-i n/Chicken Steak`     |
+|**Find in Inventory**   | `find-i [n/KEYWORDS] [s/KEYWORDS] [t/KEYWORDS]`<br> e.g., `find-i s/ntuc t/meat`     |
```

Comment:
Maybe [n/NAME] [ s/SUPPLIER] ... would be more consistent? Rather than all KEYWORDS. Same as below in line 512